setup.py

```diff
@@ -1,4 +1,6 @@
 import sys
+from io import open
```

Comment:
It may be better to add a comment here to address the python2 compatibility

Reply:
OK. I've added.